src/components/slideshow/slideshow.js

```diff
@@ -71,7 +72,8 @@ function getBackdropImageUrl(item, options, apiClient) {
     }
 
     if (item.BackdropImageTags?.length) {
-        options.tag = item.BackdropImageTags[0];
+        const randomIndex = randomInt(0, item.BackdropImageTags.length - 1);
+        options.tag = item.BackdropImageTags[randomIndex];
```

Comment:
It turned out that we also need the `index` field.
```suggestion
        options.index = randomInt(0, item.BackdropImageTags.length - 1);
        options.tag = item.BackdropImageTags[options.index];
```

Reply:
Thank you so much!